_content/booklets/discourse-on-the-snake-simile_

```diff
@@ -0,0 +1,25 @@
+---
+title: "The Discourse on the Snake Simile: Alagaddūpama Sutta with Introduction and Notes"
+translator: nyanaponika
+external_url: "https://www.accesstoinsight.org/lib/authors/nyanaponika/wheel048.html"
```

Comment:
https://www.bps.lk/olib/wh/wh048_Nyanaponika_Discourse-on-the-Snake-Simile.html  😁